modules/compliance-results.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-raw-results.adoc
+
+[id="compliance-results_{context}"]
+= Obtaining Compliance Operator raw results from a Persistent Volume
+
+.Procedure
+
+The Compliance Operator generates and stores the raw results in a Persistent Volume. These results are in Asset Reporting Format (ARF).
+
+. To view these results, explore the ComplianceSuite object:
++
+[source,terminal]
+----
+$ oc get compliancesuites nist-moderate -o json \
+| jq '.status.scanStatuses[].resultsStorage'
+{
+  "name": "rhcos4-moderate-worker",
+  "namespace": "openshift-compliance"
+}
+{
+  "name": "rhcos4-moderate-master",
+  "namespace": "openshift-compliance"
+}
+----
++
+This shows the Persistent Volume Claims where the raw results are accessible.
+
+. You can verify the raw data location by using the name and namespace of one of the results:
++
+[source,terminal]
+----
+$ oc get pvc -n openshift-compliance rhcos4-moderate-worker
+----
++
+.Example output
+[source,terminal]
+----
+NAME                 	STATUS   VOLUME                                 	CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+rhcos4-moderate-worker   Bound	pvc-548f6cfe-164b-42fe-ba13-a07cfbc77f3a   1Gi    	RWO        	gp2        	92m
+----
+
+. Fetch the raw results by spawning a pod that mounts the volume and copying the results:
++
+.Example pod
+[source,yaml]
+----
+  apiVersion: "v1"
+  kind: Pod
+  metadata:
+    name: pv-extract
+  spec:
+    containers:
+  	- name: pv-extract-pod
+    	image: registry.access.redhat.com/ubi8/ubi
+    	command: ["sleep", "3000"]
+    	volumeMounts:
+      	- mountPath: "/workers-scan-results"
+        	name: workers-scan-vol
+    volumes:
+  	- name: workers-scan-vol
+    	persistentVolumeClaim:
+      	claimName: rhcos4-moderate-worker
```

Comment:
the items int containers, volumeMounts and volumes have items with identation issues.

Reply:
Thanks, I believe this looks correct now?
```
  apiVersion: "v1"
  kind: Pod
  metadata:
    name: pv-extract
  spec:
    containers:
   	  - name: pv-extract-pod
    	  image: registry.access redhat.com/ubi8/ubi
    	  command: ["sleep", "3000"]
    	  volumeMounts:
          - mountPath: "/workers-scan-results"
            name: workers-scan-vol
    volumes:
      - name: workers-scan-vol
      	persistentVolumeClaim:
          claimName: rhcos4-moderate-worker
```